client/src/modules/Opportunities/components/ViewCv/ActionButtons/index.test.tsx

```diff
@@ -58,10 +58,10 @@ describe('ActionButtons', () => {
 
     fireEvent.click(shareButton);
 
-    const successNotification = screen.queryByText('Copied to clipboard');
+    // const successNotification = screen.queryByText('Copied to clipboard');
 
     expect(mockCopyToClipboard).not.toHaveBeenCalled();
-    expect(successNotification).toBeInTheDocument();
+    // expect(successNotification).toBeInTheDocument();
```

Comment:
plus `async` at line 54
```suggestion
    const successNotification = await screen.findByText('Copied to clipboard');

    expect(mockCopyToClipboard).not.toHaveBeenCalled();
    expect(successNotification).toBeInTheDocument();
```